src/app/routes/api/v3/admin/forms/index.ts

```diff
@@ -1 +1,9 @@
-export { AdminFormsRouter } from './admin-forms.routes'
+import { Router } from 'express'
+
+import { AdminFormsFeedbackRouter } from './admin-forms.feedback.routes'
+import { AdminFormsSettingsRouter } from './admin-forms.settings.routes'
+
+export const AdminFormsRouter = Router()
```

Comment:
use the `withUserAuthentication` middleware here, or even in the root `/admin` router.